src/test/groovy/org/prebid/server/functional/util/HttpUtil.groovy

```diff
@@ -37,4 +38,8 @@ class HttpUtil implements ObjectMapperWrapper {
     private static String encodeWithBase64(String string) {
         Base64.encoder.encodeToString(string.bytes)
     }
+
+    static String findUrlParameterValueByName(String url, String parameter) {
```

Comment:
Name too long: 6 words. Please shorten to something like `get(find)UrlParameterValue`.